src/stories/modal.stories.tsx

```diff
@@ -55,7 +55,15 @@ function ModalDemo(args) {
   )
 }
 
-const Template = (args) => {
+interface Props {
+  storyName: string
+  label: string
+  size?: "small" | "medium" | "large" | "fullscreen"
+  style?: "white" | "dark"
+  onClose: () => void
```

Comment:
is it possible to extend the components props here as well or are the Props defined explicitly here on purpose?

Reply:
The modal is not its own separate component for some reason, and it's defined within the `modal.stories.tsx` file instead of its own `modal.tsx` component file. So I'm the first to define Props for that.

![image](https://user-images.githubusercontent.com/43354234/106250922-9397d000-6214-11eb-8fc2-d88eca4aa912.png)

I didn't want to touch the components as I'm still getting into the whole project and that just creates a risk of breaking something unintentionally.